### PR TITLE
qml: add modify settings/network, add real license, fix right click, …

### PR DESCRIPTION
**Pull request by dvajdual**
…fix clicking to child items

### CODE REVIEW

File: GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        id: ip_dns_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("IP/DNS")
+
+        input_validator: RegExpValidator { regExp: /(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}(,(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3})*/ }
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: Will this validator accept host name, or only IPv4 address?
What about IPv6 address?

File: GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        id: port_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("Port")
+
+        input_validator: IntValidator {bottom: 0; top: 65536;}
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: Port number also can't be 0 and 65536, the range should be 1-65535, or better 80-65535

File: GUI/QtQuick/qml/TransactionsPage.qml [hunk 3]
```diff
@@ -168,6 +168,23 @@ Item {
...
+                    show_rbf_cfpf_menu(row, column, data)
+                }
+                else
+                {
+                    show_transaction_details(row, column, data)
+                }
+            }
+
+            function show_transaction_details(row, column, data)
+            {
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/TransactionsPage.qml]: This looks like a copy-paste from OverviewPanel. Can we have only one instance of code?

File: GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml (hunk 1 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: Only ipv4 without host

Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: Will beit ok? https://www.regextester.com/104038 Or maybe some other reg exp?

Comment (sergey-chernikov) [reply GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: Ok for IP address, but will it accept host name? We need names to be accepted, too, not only addresses.

Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml]: No will not. Ok so need remove validator

File: GUI/QtQuick/qml/TransactionsPage.qml (hunk 3 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/TransactionsPage.qml]: Yes I will create CistomTransactionsTable.qml - shared to both tables. It seems like different only width, height and model

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**